### ISSUE TITLE
Change SteVe's login username/password for administrators

### ISSUE THREAD

**Issue by anirudh-ramesh** [anirudh-ramesh/steve-docker#3]

**Comment by gabrielgruppelli** [comment]
How can I change the configurations on this project?

**Comment by anirudh-ramesh** [comment]
@gabrielgruppelli, each service in the Docker stack has a different method for configuration. Are you asking about the SteVe service?

**Comment by gabrielgruppelli** [comment]
yes, How can I change the name and password for the root, I'm not Java programmer the way you have this repo I think it is not possible to change these values, correct?

**Comment by anirudh-ramesh** [comment]
Yes, that is correct. Essentially, the Java executable file is being containerised into Docker. You need to build a new SteVe Java executable file having the chosen username and password. Building a Java executable file is outside the scope of Docker.
